From f5004522bc77f0ff54efe1786225d00903a69c2c Mon Sep 17 00:00:00 2001
From: Florin Sarbu <florin@balena.io>
Date: Wed, 4 May 2022 12:46:54 +0200
Subject: [PATCH] arm64: dts: radxa zero: Fix wifi not working after reboot

Executing a reboot will render wifi unusable due to what seems
to be a timing issue:

brcmfmac: brcmf_of_probe: interrupt could not be mapped

To fix this we change the SDIO timing to highspeed since it's
already set to it for downstream users of this bus:
https://github.com/radxa/kernel/blob/linux-5.10.y-radxa-zero/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts#L373

and

https://github.com/radxa/kernel/blob/linux-5.10.y-radxa-zero/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts#L390

Upstream-status: Pending
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
index 8306caff5b3e..0ce564a1c247 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-radxa-zero.dts
@@ -340,7 +340,7 @@
 
 	bus-width = <4>;
 	cap-sd-highspeed;
-	sd-uhs-sdr50;
+	cap-mmc-highspeed;
 	max-frequency = <100000000>;
 
 	non-removable;
-- 
2.17.1

